CONTRIBUTING.md

```diff
@@ -77,12 +77,14 @@ We use the following [labels](https://github.com/plotly/plotly.js/labels) to tra
 #### Prerequisites
 
 - git
-- [node.js](https://nodejs.org/en/). We recommend using node.js v16.x.
+- [node.js](https://nodejs.org/en/). We recommend using node.js v20.x.
```

Comment:
Node v16 should be fine. In fact that's the version we use today for most of our tests as well as the release.
Even `node v12` is still supported in our build processes and tested here: https://github.com/plotly/plotly.js/blob/8c47c16daaa2020468baf9376130e085a4f01ec6/.circleci/config.yml#L13

What was the problem you encountered with node v16?

Reply:
@marthacryan rekindling - what was the issue w/ node v16?

Reply:
I'm not sure what the issue was with node v16 but I'm no longer having it. I'll remove this change!